README.md

```diff
@@ -109,21 +129,32 @@ public interface Subscriber<T> {
 
 | ID                        | Rule                                                                                                   |
 | ------------------------- | ------------------------------------------------------------------------------------------------------ |
-| <a name="2.1">1</a>       | A `Subscriber` MUST signal demand via `Subscription.request(long n)` to receive `onNext` signals. |
-| <a name="2.2">2</a>       | If a `Subscriber` suspects that its processing of signals will negatively impact its `Publisher`'s responsivity, it is RECOMMENDED that it asynchronously dispatches its signals. |
+| <a name="2.1">1</a>       | A `Subscriber` MUST communicate demand via `Subscription.request(long n)` to receive `onNext` signals. |

```

Comment:
I think that's a fair question, and after thinking about it some more I am inclined to agree to keep it as it were. *restoring*